jscomp/build_tests/cli_help/input.js

```diff
@@ -81,7 +81,7 @@ async function test() {
     });
     assert.equal(out.stdout, buildHelp);
     assert.equal(out.stderr, "");
-    assert.equal(out.status, 0);
```

Comment:
I already fixed the tests in #6804 by changing `utils.js` to return `status` instead of `code`. So this change would break them again.

Could you undo it therefore? And also rebase to latest master to see if everything builds fine, as not all tests were actually run in CI before my recent fixes.